### PR TITLE
DOCSP-32998: Add Atlas Device SDK banner to entire project

### PR DESCRIPTION
**Pull request by cbullinger**
## Pull Request Info

**To be merged & deployed 9/26 5:55am ET**

See https://github.com/mongodb/docs-app-services/pull/587 for App Services-side PR

### Jira

- https://jira.mongodb.org/browse/DOCSP-32998

### Staged Changes

- https://preview-mongodbcbullinger.gatsbyjs.io/realm/docsp-32998-banner/

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: snooty.toml [hunk 1]
```diff
@@ -100,3 +100,11 @@ name-char-limit = "57"
 log-retention-time = "10 days"
 sync = "Atlas Device Sync"
 sync-short = "Device Sync" # feature/product name, NOT verb
+
+# Realm Rebrand announcement for .local London
+[[banners]]
+targets = ["*"]
+variant = "info"
+value = """\
+    Realm is now Atlas Device SDK – `Learn More <http://www.mongodb.com/blog/post/realm-now-part-atlas-platform>`__ \
```
Comment (cbullinger) [inline snooty.toml:109]: link currently 404's -- confirming with requester that it's correct/will be live

**Review by cbush — APPROVED** [review]
Approved for launch

File: snooty.toml (hunk 1 above)
Comment (cbullinger) [reply snooty.toml:109]: Matt confirmed blog post will go live 9/26

Comment (cbullinger) [reply snooty.toml:109]: Confirmed blog link is live

**Pull request merged**